### ISSUE TITLE
Make active-scan more aggressive (later, separately, make it configurable)

### ISSUE THREAD

**Issue by stephendonner** [stephendonner/docker-zap#3]
For now, until it's configurable, need to make the active-scan (https://github.com/stephendonner/docker-zap/blob/69c8d4853cc7ef604b811921635a10d0aa5a8ae5/run-docker.sh#L10) more aggressive against its target.

**Comment by stephendonner** [comment]
@psiinon - any immediate suggestions?  I tried reverse-engineering https://github.com/mozilla/cloudsec-zap/blob/master/amo/zap-amo-tests.sh but to little/no avail.

**Comment by psiinon** [comment]
Might be better to chat about this on vidyo - ping me on irc and we can chat about it :)

**Comment by Grunny** [comment]
If you have recommendations for which options you want, I'm happy to help add them to the `zap-cli active-scan` command as parameters (or PRs welcome!) as I'm sure other users of the docker image would find them useful. :)

**Comment by stephendonner** [comment]
At the least, I should be using the recursive -r, right, @Grunny?  Per https://github.com/Grunny/zap-cli/issues/15#issuecomment-237852475

**Comment by stephendonner** [comment]
Spoke with thc202 in #websectools (irc.mozilla.org), and added -r in with https://github.com/stephendonner/docker-zap/commit/f6068a03b31c72d8cb573804d038af06a88da65f

**Comment by Grunny** [comment]
Yep. I wonder if I should make that default and have an additional flag to not scan recursively instead.

**Comment by stephendonner** [comment]
@Grunny +1 for that!  :-)
